chart/values.yaml

```diff
@@ -0,0 +1,237 @@
+# Default values for kube-httpcache.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: quay.io/mittwald/kube-httpcache
+  pullPolicy: IfNotPresent
+  tag: "stable"
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+# Enable StatefulSet or Deployment
+statefulset:
```

Comment:
Maybe we can reduce the Statefulset/Deployment logic to a single bool field?

Something like `useStatefulset` could create a Statefulset if `true` and otherwise create a Deployment. This would prevent the creation of a Statefulset and Deployment at the same time.

Reply:
fixed